system/jlib/jiface.hpp

```diff
@@ -219,7 +234,11 @@ class CSingleThreadInterfaceOf : public CSingleThreadSimpleInterfaceOf<INTERFACE
 public:
     virtual void beforeDispose() {}
 
-    inline bool isAlive() const         { return this->xxcount < DEAD_PSEUDO_COUNT; }       //only safe if Link() is called first
+    inline bool isAliveAndLink() const
+    {
+        this->Link();
+        return this->xxcount < DEAD_PSEUDO_COUNT;
+    }
```

Comment:
does this method (and isAlive() before it), make sense given it's suppose to be SingleThreaded? Is it used?

Reply:
I think it is here so that the class can be used in the code generator without modifying any other code.